packages/components/src/common/adMixin.js

```diff
@@ -4,6 +4,14 @@ export default {
       type: Object,
       required: true,
     },
+    showMenu: {
+      type: Boolean,
+      default: true,
+    },
```

Comment:
Why do you need `showMenu` here?

Reply:
Oh, i didn't realize this got included. my bad! 😁